### PR TITLE
Add method to use Dumpling if you want to export data from TiDB

### PR DESCRIPTION
**Pull request by Tetsuya3850**
### What is changed, added or deleted? (Required)
In the migration FAQ, dumpling was not listed as a way to export data from TiDB. 
I belive this is the recommended method than mysqldump. So I added it.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Tetsuya3850** [comment]
@qiancai Hi, much appreciated if you can please take a look at your convenience 🙇

**Comment by Tetsuya3850** [comment]
@ran-huang Hi, much appreciated if you can please take a look at your convenience 🙇

File: faq/migration-tidb-faq.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ Restart the TiDB service, add the `-skip-grant-table=true` parameter in the conf
 
 You can use the following methods to export the data in TiDB:
 
+- Export data using Dumpling. For more details refer [here](/dumpling-overview.md)
```
Comment (qiancai) [inline faq/migration-tidb-faq.md:81]:
```suggestion
- Export data using Dumpling. For more information, see [Dumpling documentation](/dumpling-overview.md).
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**